ibis/backends/sql/compilers/pyspark.py

```diff
@@ -589,3 +594,43 @@ def _format_window_interval(self, expression):
         this = expression.this.this  # avoid quoting the interval as a string literal
 
         return f"{this}{unit}"
+
+    def visit_ArraySumAgg(self, op, *, arg, output):
```

Comment:
OMG hahah wow pyspark is so needy here. Duckdb team needs to use this for their talking point of "ergonomic SQL"

Reply:
Yeah, this one was a doozy! TBF Trino also requires this kind of special treatment, because using the fully general `reduce` (Trino) or `aggregate` (Spark) requires handling null behavior.